src/SFA.DAS.EAS.Portal.Client/Services/DasRecruit/DasRecruitService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Client.Http;
+using SFA.DAS.EAS.Portal.Client.Services.DasRecruit.Models;
+using SFA.DAS.EAS.Portal.Client.Types;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EAS.Portal.Client.Services.DasRecruit
+{
+    /// <summary>
+    /// https://skillsfundingagency.atlassian.net/wiki/spaces/RAAV2/pages/200245289/Environments
+    /// </summary>
+    internal class DasRecruitService : IDasRecruitService
+    {
+        private readonly ILog _log;
+        private readonly HttpClient _httpClient;
+        private readonly Type _vacancyStatusType;
+
+        public DasRecruitService(IRecruitApiHttpClientFactory recruitApiHttpClientFactory, ILog log)
+        {
+            _log = log;
+            //todo: think through lifetimes
+            _httpClient = recruitApiHttpClientFactory.CreateHttpClient();
```

Comment:
Should httpClient be injected in as a singleton as per here?

 https://docs.microsoft.com/en-gb/dotnet/api/system.net.http.httpclient?view=netframework-4.7.1

Reply:
That's what the `todo` was there for, to make sure that the HttpClient is a singleton 😄.

I've made the `DasRecruitService` a singleton, which effectively makes `HttpClient` a singleton.

Reply:
👍